.github/workflows/test.yml

```diff
@@ -13,44 +13,53 @@ jobs:
     strategy: 
       matrix:
         os: [macos-11, macos-12, macos-13, ubuntu-20.04, ubuntu-22.04, windows-2019, windows-2022]
```

Comment:
```suggestion
        os: [macos-latest, ubuntu-latest, windows-latest]
```

Reply:
check https://github.com/neo-project/neo-express/blob/master/.github/workflows/push.yml do you still want it to publish to github?